staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest_test.go

```diff
@@ -326,6 +327,9 @@ func (tc *patchTestCase) Run(t *testing.T) {
 		patch := []byte{}
 		switch patchType {
 		case types.StrategicMergePatchType:
+			if len(tc.jsonPatch) > 0 {
+				continue
+			}
```

Comment:
Are these lines for debugging only? If so please remove.

Reply:
Done.